docs/source/dashboard.md

```diff
@@ -0,0 +1,141 @@
+
+# Dashboard
+
+If you want to compare several solvers on a bunch of problem instances, we have you cover with the discrete-optimization dashboard!
+
+This tool is used to summarize in a few graphs your experiments.
+The following steps are necessary:
+- Create a study by running different solver *configs* on several problem *instances*.
+- Store time-indexed metrics for each experiment like d-o fitness, internal model objective, bound, gap, ...,
+  with the necessary metadata (solver config, problem instance, solver final status).
+- Retrieve this data as a list of pandas dataframes.
+- Launch the dashboard to visualize
+
+We enter into details in the next sections.
+
+![dashboard in action](dashboard.gif)
+
+## Prerequisites
+
+For the dashboard to run you need to install "dash", "plotly", and "dash_bootstrap_components" libraries.
+If you want to make use of the hdf5 database presented below to store the data, you will also need "pytables".
+All these libraries can be installed by choosing the extra "dashboard" when installing discrete-optimization:
+```shell
+pip install discrete-optimization[dashboard]
+```
+
+
+## Launching a d-o study
+
+We call here a study a set of experiments, where each experiment is the choice of
+- a solver config (class and hyperparameters)
+- a problem instance to solve
+
+### Experiment
```

Comment:
for experiment and stats retriever callbacks, i would be interested to get code snippets examples, actually showing how to launch a study.
code snippet a bit like you put on other subsections.